### ISSUE TITLE
Adblock

### ISSUE THREAD

**Issue by TulioPro** [photopea/photopea#3553]
I disabled adblock the first time i used photopea but now when i wanna erase something it appears me "We use advertisement to get money for our development. Your device seems to be blocking ads. Please, let us show you ads, or buy a Premium account (Account - Log In - Buy).  5" but i disabled adblock

**Comment by inayyu** [comment]
i have the same problem.

**Comment by AvinashGudidevini** [comment]
How do I solve this issue for Adblock? Cab anyone guide me through it!

**Comment by Jousw** [comment]
I Don't Even Have Any Adblock Enabled I Don't Know Why I Have To Deal With This
![Photopea](https://user-images.githubusercontent.com/87468448/125751632-f02d3adb-69bd-4852-84d8-b8b37ee2ad49.png)

**Comment by MichalSZZ** [comment]
Do you see any ads on the right side?
If no - that means you are using some software to block advertisements or someone else is blocking it for you.
Even if you think that you are not using any there can be many possible things: 

- You are using Browser that have build in adblocker (for example Brave)
- You have antivirus/firewall program that block ads as a built in feature
- You have some other software installed on your device for keeping your privacy
- Your computer administrator did something (in your work or school) and you can't even see that 
- Your internet provider can block some files from specific domains
- You have WiFi router / modem that has build in ads free / privacy feature
- You have some virus on your computer or router, that puts in they own advertisements on sites
- You have some other addon for your browser that you think is is not responsible for blocking ads but it change some source code and can add cookies or something else
- Your AdBlock even if you turn off advertisements on the photopea.com, still puts some other things like cookies

**Comment by MichalSZZ** [comment]
And finally - I am a Photopea user since 4 years. I didn't have such a info ever. I use many browsers and computers. Sometimes I am logged in sometimes no. Sometimes I payed for photopea but mostly - no. 
If I had unpayed account I see every time ads on the right side. 

But I don't have any extension in my Chrome where I use Photopea.
For any other things I use Firefox and there I have one or two extensions.
If you like for example Brave but want to use Photopea for free without any problems - use clear Chrome or clear and newest Edge browser. There are functionality to "install" Photopea as a PWA application.

And...
I am a normal user - not Photopea worker. And I can't watch all the time here or on Reddit people who want to use that amazing software without seeing ads or paying for it. Author of the program have to get money for his very hard work. Now from a week or more he only answers on such posts like this... I like when he is coding new wonderful things to Photopea - and you?
